openmls/tests/test_managed_group.rs

```diff
@@ -84,6 +84,33 @@ fn auto_save(managed_group: &ManagedGroup) {
         .expect("Could not write group state to file");
 }
 
+// The following enables the OpenMlsEvercrypt provider on machines that support
+// it. This is a basic check to ensure that the provider generally works.
+// TODO: #520 - Better tests for Evercrypt backend
+#[cfg(all(
+    target_arch = "x86_64",
+    not(target_os = "macos"),
```

Comment:
Any reason we exclude `macOS` currently?

Reply:
The Github CI doesn't support AES-NI on macOS. This should change with the new runners in about a month.